src/ui/bind_handlers.js

```diff
@@ -177,6 +177,10 @@ export default function bindHandlers(map: Map, options: {interactive: boolean})
     }
 
     function onWheel(e: WheelEvent) {
+        if (options.interactive) {
+            map.stop();
+        }
+
```

Comment:
Is this related to the resize fix or something separate?
No need to split it out if it's separate, I'm just wondering if I'm missing a relation between this and the resize bug.

Reply:
I think I needed this so that scrolling on the map cancelled an active flyTo.